clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/ReconnectHandle.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ehcache.clustered.client.internal.store;
+
+public interface ReconnectHandle {
```

Comment:
Should we just use `Runnable` here?

Reply:
Oh it is still lingering around here, I removed it though. :confused: